e2e/playwright/tests/@version-history-pagination/test.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { test, expect, Page } from '@playwright/test';
+import { login, uploadLicense } from '../shared';
+import { execSync } from 'child_process';
+
+test('version history pagination', async ({ page }) => {
+  test.slow();
+  test.setTimeout(300000); // 300 seconds, 5 minutes
+  const testAppSlug = "version-history-pagination";
+  const testNamespace = "version-history-pagination";
+  const testNumOfVersions = 251;
+  const testLatestSequence = 251;
+  const testDefaultPageSize = 20;
+  await login(page);
+  await uploadLicense(page, expect, "version-history-pagination.yaml");
+  await expect(page.locator('#app')).toContainText('Configure Version History Pagination', { timeout: 15000 });
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await expect(page.locator('#app')).toContainText('Currently deployed version', { timeout: 15000 });
+  await expect(page.locator('#app')).toContainText('Ready', { timeout: 30000 });
```

Comment:
there's an existing `dashboard-app-status` data-testid for the app status:
```suggestion
  await expect(page.getByTestId("dashboard-app-status")).toHaveText("Ready", { timeout: 30000 });
```

Reply:
edited